examples/auth-generate-jwt/javascript/src/styles.css

```diff
@@ -1,100 +1 @@
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
-
-.container {
-  width: 100%;
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  position: relative;
-  background-color: #f4f8fb;
-  height: 100vh;
-}
-
-input {
-  padding: 0.5rem;
-  width: 100%;
-  height: 2.5rem;
-  font-size: 0.875rem;
-  border-radius: 0.375rem;
-  outline: none;
-  transition: all 0.2s ease-in-out;
-}
-
-@layer base {
-  :root {
-    --background: 0 0% 100%;
-    --foreground: 20 14.3% 4.1%;
-    --card: 0 0% 100%;
-    --card-foreground: 20 14.3% 4.1%;
-    --popover: 0 0% 100%;
-    --popover-foreground: 20 14.3% 4.1%;
-    --primary: 20 91% 54%;
-    --primary-foreground: 60 9.1% 97.8%;
-    --secondary: 209 100% 43%;
-    --secondary-foreground: 60 9.1% 97.8%;
-    --muted: 60 4.8% 95.9%;
-    --muted-foreground: 25 5.3% 44.7%;
-    --accent: 60 4.8% 95.9%;
-    --accent-foreground: 24 9.8% 10%;
-    --destructive: 0 84.2% 60.2%;
-    --destructive-foreground: 60 9.1% 97.8%;
-    --border: 20 5.9% 90%;
-    --input: 20 5.9% 90%;
-    --ring: 24.6 95% 53.1%;
-    --radius: 0.5rem;
-    --chart-1: 12 76% 61%;
-    --chart-2: 173 58% 39%;
-    --chart-3: 197 37% 24%;
-    --chart-4: 43 74% 66%;
-    --chart-5: 27 87% 67%;
-  }
-
-  .dark {
-    --background: 20 14.3% 4.1%;
-    --foreground: 60 9.1% 97.8%;
-    --card: 20 14.3% 4.1%;
-    --card-foreground: 60 9.1% 97.8%;
-    --popover: 20 14.3% 4.1%;
-    --popover-foreground: 60 9.1% 97.8%;
-    --primary: 20 91% 54%;
-    --primary-foreground: 60 9.1% 97.8%;
-    --secondary: 209 100% 43%;
-    --secondary-foreground: 60 9.1% 97.8%;
-    --muted: 12 6.5% 15.1%;
-    --muted-foreground: 24 5.4% 63.9%;
-    --accent: 12 6.5% 15.1%;
-    --accent-foreground: 60 9.1% 97.8%;
-    --destructive: 0 72.2% 50.6%;
-    --destructive-foreground: 60 9.1% 97.8%;
-    --border: 12 6.5% 15.1%;
-    --input: 12 6.5% 15.1%;
-    --ring: 20.5 90.2% 48.2%;
-    --chart-1: 220 70% 50%;
-    --chart-2: 160 60% 45%;
-    --chart-3: 30 80% 55%;
-    --chart-4: 280 65% 60%;
-    --chart-5: 340 75% 55%;
-  }
-}
-
-.container {
-  width: 100%;
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  position: relative;
-  background-color: #f4f8fb;
-  height: 100vh;
-}
-
-input {
-  padding: 0.5rem;
-  width: 100%;
-  height: 2.5rem;
-  font-size: 0.875rem;
-  border-radius: 0.375rem;
-  outline: none;
-  transition: all 0.2s ease-in-out;
-}
+@import '../../../styles.css';
```

Comment:
This is an indicator of a change common throughout this PR - project stylesheets have been de-duplicated, with `examples/styles.css` housing all the common styles, and these ones only importing and adding unique styles.

In the case of no unique styles, the React apps import from `examples/styles.css` directly. The JS ones have a one line file with an input since `index.html` can't seem to reach back beyond the directory it's in.

Reply:
@jamiehenson how will Sandpack handle this? maybe we can include the file and rewrite the import statement while passing it in?

Reply:
@kennethkalmer I've bitten the bullet and brought all the stylesheets back in for the react projects, also relocating them into `/src` which sandpack more natively expects.

Squashed and cherry picked this into the sandpack branch, it works 👍 (thankfully, the child postcss/vite/tw/ts configs are minimal enough to be handled by sandpack's defaults it seems - so we can maintain a compromise between de-duplication in the repo and stuff that works downstream)

Reply:
One dumb thing I was bashing my head against though is that Sandpack hates Tailwind if it's not brought in as an external script - but that's a downstream change, not for here